app/tasks.py

```diff
@@ -82,7 +102,7 @@ def parseDate(date_f):
     # fileObject is your csv.reader
     total_row_count = sum(1 for line in csv.reader(csvfile)) - 1
```

Comment:
We should consider using csv.DictReader later. I didn't know it was a thing before.